examples/vision/video_classification.py

```diff
@@ -37,28 +43,39 @@
 subsampled version of the original UCF101 dataset. You can refer to
 [this notebook](https://colab.research.google.com/github/sayakpaul/Action-Recognition-in-TensorFlow/blob/main/Data_Preparation_UCF101.ipynb)
 to know how the subsampling was done.
+
+#### Note
+
+There was an issue with the link to the dataset, and it was resolved, but at this moment the dataset generation
+notebook is not entirely compatible with this working version of the example, see: https://github.com/keras-team/keras-io/issues/1342
 """
```

Comment:
As long as users can run this example, I don't think there's a need to add this section.

Reply:
I just wanted to mention that your original notebook for data preparation would have to be modified a bit to generate exactly the dataset that is used now in the video classification example. Ideally would be to adjust/prepare the data preparation notebook as well, and either include it as a Keras example, or just provide a link to the colab/github resource where the updated notebook would be available. If I will find a minute, I'll try to prepare the notebook. Meanwhile I can remove the "Note" from the video classification example, to avoid confusion.

Reply:
I removed this portion, as you asked.